examples/all-clusters-app/all-clusters-common/src/laundry-washer-controls-delegate-impl.cpp

```diff
@@ -54,3 +55,44 @@ CHIP_ERROR LaundryWasherControlDelegate::GetSupportedRinseAtIndex(size_t index,
     supportedRinse = LaundryWasherControlDelegate::supportRinsesOptions[index];
     return CHIP_NO_ERROR;
 }
+
+Status LaundryWasherControlDelegate::PreAttributeCheck(AttributeId attributeId, uint16_t size, uint8_t * value)
+{
+    switch (attributeId)
+    {
+    case Attributes::SpinSpeedCurrent::Id: {
+        if (*value >= ArraySize(spinSpeedsNameOptions))
+        {
+            return Status::ConstraintError;
+        }
+        return Status::Success;
+    }
+    case Attributes::NumberOfRinses::Id: {
+        if (size != sizeof(NumberOfRinsesEnum))
```

Comment:
There was no size check for SpinSpeedCurrent.  Why is that OK if we need a size check here?

Reply:
It should not be required. And I will move it to the cluster handler